Automattic/simplenote-android
# Issue/update sheet buttons

### Fix
Update the buttons in the history and info bottom sheets.

#### History
The buttons in the history bottom sheet, _**Cancel**_ and _**Restore**_, had a `selectableItemBackgroundBorderless` background, which doesn't follow the guidelines for [text buttons in Material documentation](https://material.io/design/components/buttons.html#text-button).  The background was updated to `selectableItemBackground`, which mimics the guidelines as shown [here](https://storage.googleapis.com/spec-host-backup/mio-design%2Fassets%2F14c0HqGVluF8KDMuij_AV5puYPmorTU2u%2Ftext-buttons-states.png).  See the screenshots below for illustration.

![simplenote_history_buttons](https://user-images.githubusercontent.com/3827611/50402126-275b6000-0762-11e9-86f1-44bb48ab5f8b.png)

#### Info
The share button in the info bottom sheet was missing an icon and the view width was set to `match_parent`, which made the tapped background fill the bottom sheet.  Also, the view's content description was set to `@string/copy`, which incorrectly read as "Copy" for the share button.  The view's width, content description, and icon were updated.  Also, a hint and haptic feedback were added to both the share and copy buttons when long-pressed.  See the screenshots below for illustration.

![simplenote_info_compare](https://user-images.githubusercontent.com/3827611/50402227-dac45480-0762-11e9-8fbf-51a7c64cbfa0.png)

![simplenote_info_all](https://user-images.githubusercontent.com/3827611/50402228-dd26ae80-0762-11e9-9cdb-a280ceb8efbe.png)

### Test
1. Tap unpublished note in list.
2. Tap **_History_** action in toolbar.
3. Long-press **_Cancel_** button in bottom sheet.
4. Notice button background as shown above.
5. Long-press **_Restore_** button in bottom sheet.
6. Notice button background as shown above.
7. Tap **_Info_** action in toolbar.
8. Notice **_Share_** button icon as shown above.
9. Long-press **_Share_** button in bottom sheet.
10. Notice button background as shown above.
11. Notice **_Share_** hint and haptic feedback.
12. Tap **_Share_** button in bottom sheet.
13. Tap **_Publish_** button in sharing sheet.
14. Wait for note to be published.
15. Tap **_Info_** action in toolbar.
16. Long-press **_Copy_** button in bottom sheet.
17. Notice button background as shown above.
18. Notice **_Copy_** hint and haptic feedback.
